private/mplgndr.m

```diff
@@ -0,0 +1,72 @@
+function plgndr = mplgndr ( degree, order, x )
+
+% MPLGNDR associated Legendre functions
+%
+% y = mplgndr(n,k,x) computes the values of the associated Legendre
+% functions of order K up to degree N.
+% 
+% The input x can be a matrix, and the result is of size numel(x) by N+1.
+% The i-th column is the associated Legendre function of order K and
+% degree i-1.
+
+% Copyright (C) 2002, Robert Oostenveld
+% Copyright (C) 2021, Ricardo Bruña
+% 
+% This file is part of FieldTrip, see http://www.fieldtriptoolbox.org
+% for the documentation and details.
+%
+%    FieldTrip is free software: you can redistribute it and/or modify
+%    it under the terms of the GNU General Public License as published by
+%    the Free Software Foundation, either version 3 of the License, or
+%    (at your option) any later version.
+%
+%    FieldTrip is distributed in the hope that it will be useful,
+%    but WITHOUT ANY WARRANTY; without even the implied warranty of
+%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%    GNU General Public License for more details.
+%
+%    You should have received a copy of the GNU General Public License
+%    along with FieldTrip. If not, see <http://www.gnu.org/licenses/>.
+%
+% $Id$
+
+
+% Checks the input.
+if order < 0 || round ( order ) ~= order
```

Comment:
Could I ask you to update this file to have 2 spaces for a tab, rather than 4. 
Robert is quite keen on having this consistent.

This comment also counts for the other new function

Reply:
Of course. I do prefer 2 spaces, in fact, 4 is the default Matlab.

Do you know if I can change it? It would be great...

Reply:
Done

Reply:
In the matlab preferences you can specificy the number of spaces per tab:

environment -> preferences -> editor/debugger -> tab